docs/src/userguide/language_basics.rst

```diff
@@ -45,61 +48,104 @@ C variable and type definitions
 ===============================
 
 The :keyword:`cdef` statement is used to declare C variables, either local or
-module-level::
+module-level
```

Comment:
This statement is incorrect for the Python version. Also, since type hints are ignored for global variables, this needs to use the explicit `cython.declare()` command.

Reply:
> this needs to use the explicit cython.declare() command.

It would be very good to indicate this, for example in a comment or explanation, since this behavior has not yet been documented, although in a good way it belongs [here](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html).

Reply:
In addition, you can make the pure python version match the version with `cdef` just with `cython.declare()`.
i.e. `cython.declare(i=cython.int, j=cython.int, k=cython.int)`

and for alignment reasons, `import cython` is not needed here, especially since it has already been written about in `two-syntax-variants-used`

Reply:
done.